Frameworks/CoreText/DWriteWrapper_CTFont.mm

```diff
@@ -358,5 +360,41 @@ CFStringRef _DWriteFontCopyName(const ComPtr<IDWriteFontFace>& fontFace, CFStrin
         informationalStringId = DWRITE_INFORMATIONAL_STRING_NONE;
     }
 
-    return _DWriteFontCopyInformationalString(fontFace, informationalStringId);
+    return _DWriteFontCopyInformationalString(fontFace, informationalStringId, actualLanuguage);
+}
+
+static constexpr CTFontTableTag sc_tags[] = { kCTFontTableBASE, kCTFontTableCFF,  kCTFontTableDSIG, kCTFontTableEBDT, kCTFontTableEBLC,
+                                              kCTFontTableEBSC, kCTFontTableGDEF, kCTFontTableGPOS, kCTFontTableGSUB, kCTFontTableJSTF,
+                                              kCTFontTableLTSH, kCTFontTableOS2,  kCTFontTablePCLT, kCTFontTableVDMX, kCTFontTableVORG,
+                                              kCTFontTableZapf, kCTFontTableAcnt, kCTFontTableAvar, kCTFontTableBdat, kCTFontTableBhed,
+                                              kCTFontTableBloc, kCTFontTableBsln, kCTFontTableCmap, kCTFontTableCvar, kCTFontTableCvt,
+                                              kCTFontTableFdsc, kCTFontTableFeat, kCTFontTableFmtx, kCTFontTableFpgm, kCTFontTableFvar,
+                                              kCTFontTableGasp, kCTFontTableGlyf, kCTFontTableGvar, kCTFontTableHdmx, kCTFontTableHead,
+                                              kCTFontTableHhea, kCTFontTableHmtx, kCTFontTableHsty, kCTFontTableJust, kCTFontTableKern,
+                                              kCTFontTableKerx, kCTFontTableLcar, kCTFontTableLoca, kCTFontTableMaxp, kCTFontTableMort,
+                                              kCTFontTableMorx, kCTFontTableName, kCTFontTableOpbd, kCTFontTablePost, kCTFontTablePrep,
+                                              kCTFontTableProp, kCTFontTableSbit, kCTFontTableSbix, kCTFontTableTrak, kCTFontTableVhea,
+                                              kCTFontTableVmtx };
+
+CFArrayRef _DWriteCopyAvailableFontTables(const ComPtr<IDWriteFontFace>& fontFace) {
+    auto ret = woc::MakeAutoCF<CFMutableArrayRef>(CFArrayCreateMutable(nullptr, 0, nullptr));
+    for (CTFontTableTag tag : sc_tags) {
+        const void* tableData;
+        uint32_t tableSize;
+        void* tableContext;
+        BOOL exists;
+
+        RETURN_NULL_IF_FAILED(fontFace->TryGetFontTable(_CTToDWriteFontTableTag(tag), &tableData, &tableSize, &tableContext, &exists));
+        if (exists) {
+            // Returned array is expected to have unboxed CTFontTableTag values
+            CFArrayAppendValue(ret, (const void*)tag);
```

Comment:
This array isn't using any memory management callbacks so it shouldn't be leaking anything, but it's worth checking.